fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+	font-family: 'PT Serif';
+	src: url('fonts/ptserif-bold-webfont.woff') format('woff');
+	font-weight: 700 ;
+	font-style: normal;
+}
+
+@font-face {
+  font-family: 'Roboto';
+	src: url('fonts/roboto-bold-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/second_final_np/pullRequest?prid=6218911)